datalab/datalab_session/util.py

```diff
@@ -148,15 +156,26 @@ def get_hdu(basename: str, extension: str = 'SCI', source: str = 'archive') -> l
   
   return extension
 
+def fits_dimensions(fits_file) -> tuple:
+  hdu = fits.open(fits_file)
+  height, width = hdu[1].shape
+  return height, width
+
 def create_fits(key: str, image_arr: np.ndarray) -> fits.HDUList:
+  """
+  Creates a fits file with the given key and image array
+  Returns the HDUList and the dimensions of the image
```

Comment:
This function doesn't appear to return an HDUList and the dimensions, it returns a path?